CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/BottomButtonsViewModel.swift

```diff
@@ -8,12 +8,12 @@
 
 import Foundation
 
-public final class BottomButtonsViewModel {
```

Comment:
[Sugesstion] I am afraid that we would need to keep it public. This class for the model will be reused in error screen as well 🧐